deep-learning/_posts/2018-07-10-Spcnn.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: review
+title:  "Deep Networks With Shape Priors for Nucleus Detection"
+tags:   Biomedical Segmentation
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1807.03135.pdf   
+cite:
+authors: "Mohammad Tofighi, Tiantong Guo, Jairam K.P. Vanamala, Vishal Monga"
+title:   "Deep Networks With Shape Priors for Nucleus Detection"
+venue:   "arxiv"
+---
+
+### Description
+
+This paper proposes a network structure that can incorporate ‘expected behavior’ of nucleus shapes via two components: learnable layers that perform the nucleus detection and a fixed processing part that guides the learning with prior information which can be obtained in consultation with a medical domain expert. A set of shape priors developed with the help of domain expert to hand label the nuclei boundaries as shown in fig2. Ideally, the network's segmentation prediction should lie inside of these nuclei boundaries (prior shapes).
+
+<img src="/deep-learning/images/spcnn/0.png" width="400">
+
+Three sources contribute to generating the prior: network output, raw edge map from the input image $hat{x}$ which obtained by Canny edge detection filter,  and a set of predefined shapes prepared $S$ by the medical expert.
```

Comment:
"[...] which **is** obtained by [...]"
"[...] and a set of predefined shapes **$S$** prepared by [...]"